zone/zone_loot.cpp

```diff
@@ -5,20 +5,36 @@
 #include "../common/repositories/lootdrop_repository.h"
 #include "../common/repositories/lootdrop_entries_repository.h"
 
-void Zone::LoadLootTables(const std::vector<uint32> &loottable_ids)
```

Comment:
And this reference, why did we remove it?

Reply:
We modify it by filtering on it below, it wouldn't work on the reference so I made a copy. Very low overhead operation to do the copy anyhow